sagas/apiCitiesSaga.js

```diff
@@ -0,0 +1,13 @@
+import { call, put, takeEvery } from 'redux-saga/effects';
+import ApiHelper from '../pages/ApiHelper';
+import { fetchCitiesFromAPI, addCities } from '../actions/apiCitiesAction';
+
+function* fetchCities() {
+  console.log('In saga..........');
+  const data = yield call(ApiHelper, 'cities');
+  yield put(addCities(data));
```

Comment:
yield put(actionName()) required in every saga. can we move that to apiHelper saga.